app/models/order.rb

```diff
@@ -125,6 +125,21 @@ def init_dates
     self
   end
 
+  def self.map_with_own_group_orders(scope, ordergroup, page = nil, limit = nil)
```

Comment:
IMOH `own` isn't very clear here. what about `ordergroup_group_orders_map`?

Reply:
Okay, I'll change the `own` to `ordergroup` everywhere - I just needed a word for "the Ordergroup associated with the logged in User".